File: Programmers/Level1/최대공약수와최소공배수.py
def solution(n, m):
    answer = []
    for i in range(1, n+1):
        if n % i == 0:
            if m % i == 0:
                gcd = i
    if gcd == 1:
        lcm = n*m
    else:
        lcm = gcd * (n // gcd) * (m // gcd)
    answer.extend([gcd, lcm])
    return answer

n = 3
m = 12
print(solution(n, m))

File: Programmers/Level1/최대공약수와최소공배수/최대공약수와최소공배수.js
function solution(n, m) {
  let gcd = 0;
  let lcm = 0;
  for (let i = 1; i <= n; i++) {
    if (!(n % i) && !(m % i)) {
      gcd = i;
    }
  }

  if (gcd === 1) {
    lcm = n * m;
  } else {
    lcm = gcd * parseInt(n / gcd) * parseInt(m / gcd);
  }
  return [gcd, lcm];
}

const [n, m] = [2, 5];

console.log(solution(n, m));
